SendLog.py

```diff
@@ -1,28 +1,37 @@
-## Hrishi Hiraskar
-## 23 October 2016
-
+from __future__ import print_function
+from xml.dom import minidom
 import gevent
+import fileinput
 import time
+import shutil
 import os
+import time
 import threading
 from gevent import monkey
+import fileinput
 from gevent.pywsgi import WSGIServer
-from flask import Flask, request, Response, render_template, send_from_directory
+from flask import Flask, request, Response, render_template, send_from_directory ,send_file# send_file added to ease download
 from werkzeug import secure_filename
+from os.path import exists
+#import webbrowser #modifiedm@shivendra for displaying image saved
+#from random import randint # modified_shank : to generate random image names
 
 monkey.patch_all(aggressive=False)
 
-import subprocess
+import subprocess32 as subprocess
```

Comment:
why are you using a module written for python3 and backported to python2? why not use subprocess itself? This is risky.